### ISSUE TITLE
Frame round corner problem

### ISSUE THREAD

**Issue by zemian** [Spiderpig86/Cirrus#191]
Hi, when enlarging the round corner of a frame, there is an issue where the bottom corners background is showing different colors. Is there way to fix this? 

Below is a test case:

![image](https://github.com/Spiderpig86/Cirrus/assets/22258832/488f8c25-dd08-4aa3-8a6f-04c9edbc711f)

```
<div class="min-h-100p bg-gray-200">
   <section>
      <div class="content px-12 py-4">
         <div class="w-32">
            <div class="frame u-round-lg u-shadow-lg bg-purple-500">
               <div class="frame__body py-8"></div>
               <div class="frame__footer bg-white py-1 u-bg-opacity-80">
                  <div class="text-gray-700 text-sm"><b>purple-500</b></div>
               </div>
            </div>
         </div>
      </div>
   </section>
</div>
```

**Comment by Spiderpig86** [comment]
Ack will take a look

**Comment by Spiderpig86** [comment]
You can use the `u-overflow-hidden` class hide the white background.

```html
<div class="min-h-100p bg-gray-200">
   <section>
      <div class="content px-12 py-4">
         <div class="w-32">
            <div class="frame u-round-lg u-shadow-lg bg-purple-500 u-overflow-hidden">
               <div class="frame__body py-8"></div>
               <div class="frame__footer bg-white py-1 u-bg-opacity-80">
                  <div class="text-gray-700 text-sm"><b>purple-500</b></div>
               </div>
            </div>
         </div>
      </div>
   </section>
</div>
```

**Comment by zemian** [comment]
Ah, great solution! Thanks!

BTW, I want to point out that the docs on "colors" does not use the fix above, so it shows up on all the Extended colors pallet examples. (The round is small, so one would need to look closely to see it). So perhaps the doc should be fixed? Or even better is that the "Frame" should have this fix as default, since it has the rounded corner?

**Comment by Spiderpig86** [comment]
This has been resolved in commit 7fd9209c1488a2f91dd82a6c6a8c00dcdebc5bd2 and will be shipped in 0.7.2
